src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -112,7 +112,7 @@ public bool MatchesPattern(string text, string pattern, CultureInfo culture)
             if (!culture.Equals(EnUSCultureInfo))
             {
                 patternMatcher = this.GetPatternMatcher(pattern, EnUSCultureInfo);
-                match = patternMatcher.GetFirstMatch(completionItemText);
+                match = patternMatcher.GetFirstMatch(completionItemText, includeMatchSpans);
```

Comment:
@CyrusNajmabadi When you removed the default value from the parameter in `GetFirstMatch`, it should have been inlined here with its previous default value, `false`. You instead passed through a different value. If the new value is actually the correct value, that correction is unrelated to the rest of this pull request and needs its own separate bug/test.